Tools/eslint-config-cesium/browser.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = {
+    'extends': 'cesium',
```

Comment:
whoops- I did this locally but did not push. I'll push all the new changes soon.